lib/va_profile/profile_information/service.rb

```diff
@@ -0,0 +1,201 @@
+# frozen_string_literal: true
+
+require 'common/client/concerns/monitoring'
+require 'common/client/errors'
+require 'va_profile/service'
+require 'va_profile/stats'
+require_relative 'configuration'
+require_relative 'transaction_response'
+require_relative 'person_response'
+
+module VAProfile
+  module ProfileInformation
+    class Service < VAProfile::Service
+      include Common::Client::Concerns::Monitoring
+
+      STATSD_KEY_PREFIX = "#{VAProfile::Service::STATSD_KEY_PREFIX}.profile_information".freeze
+      configuration VAProfile::ProfileInformation::Configuration
+
+      OID = MPI::Constants::VA_ROOT_OID
+      AAID = '^NI^200DOD^USDOD'
+
+      CONTACT_INFO_CHANGE_TEMPLATE = Settings.vanotify.services.va_gov.template_id.contact_info_change
+      EMAIL_PERSONALISATIONS = {
+        address: 'Address',
+        residence_address: 'Home address',
+        correspondence_address: 'Mailing address',
+        email: 'Email address',
+        phone: 'Phone number',
+        home_phone: 'Home phone number',
+        mobile_phone: 'Mobile phone number',
+        work_phone: 'Work phone number'
+      }.freeze
+
+      # def get_response(type)
```

Comment:
Can this be removed as well? We always have git history if it needs to be resurrected

Reply:
Yeah, i can remove it for now :)